tests/FsCheck.Test/project.json

```diff
@@ -0,0 +1,44 @@
+{
+  "version": "1.0.0-*",
+  "buildOptions": {
+    "debugType": "portable",
+    "compilerName": "fsc",
+    "compile": {
+      "includeFiles": [
+        "Helpers.fs",
+        "Random.fs",
+        "Property.fs",
```

Comment:
these are the issue about boostrapping :D we just need some few lib more
It's the same with other projects who use fscheck for tests too :D